src/Sylius/Behat/Service/SlugGenerationHelper.php

```diff
@@ -29,11 +28,11 @@
      */
     public static function waitForSlugGeneration(Session $session, NodeElement $element)
     {
-        Assert::isInstanceOf($session->getDriver(), Selenium2Driver::class);
+        Assert::true(DriverHelper::supportsJavascript($session->getDriver()), 'Browser does not support Javascript.');
 
         JQueryHelper::waitForAsynchronousActionsToFinish($session);
-        static::isElementReadonly($session, $element);
-        JQueryHelper::waitForAsynchronousActionsToFinish($session);
+
+        usleep(500000); // TODO: Remove hardcoded sleep from tests
```

Comment:
Actually, it looks like `static::isElementReadonly($session, $element)` haven't been workng correctly for some cases and it was waiting for 5 seconds, here we make it a stable 0.5s :)